demo/streaming_stt/run.py

```diff
@@ -0,0 +1,43 @@
+from deepspeech import Model
```

Comment:
Suggestion: replace this with a ASR model from `transformers`. I can do this tomorrow